src/ArticleContent.jsx

```diff
@@ -38,19 +57,32 @@ export function ArticleContent() {
             });
     };
 
-    useEffect(() => {
+    const handleCommentSubmit = (event) => {
+        event.preventDefault();
+        const commentBody = event.target.elements.comment.value;
+
+        setIsPostingComment(true);
         axios
-            .get(`https://xqnews.onrender.com/api/articles/${article_id}`)
+            .post(
+                `https://xqnews.onrender.com/api/articles/${article_id}/comments`,
+                {
+                    username: user.username,
+                    body: commentBody,
+                }
+            )
             .then((response) => {
-                setArticleContent(response.data.article);
-                setArticlePoints(response.data.article.votes);
-                setIsLoading(false);
+                setCommentList([response.data.comment, ...commentList]);
+                event.target.reset();
+                setCommentFeedback("Your comment has been posted!");
             })
             .catch((error) => {
-                setError("There was an error fetching the article content!");
-                setIsLoading(false);
+                setError("There was an error posting the comment.");
+                setCommentFeedback("");
```

Comment:
There is an error being set, but I don't see this on the page at all? It would be great to have a message being shown in the case of a failed comment post, so the user knows it failed and they aren't confused.